spacy/tests/lang/lb/test_lemma.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "string,lemma",
+    [
+        ("Dëscher", "Dësch"),
+        ("engagéiers", "engagéieren"),
+        ("goung", "goen"),
+        ("neit", "nei"),
+        ("déi", "déi"),
+        ("Männer", "Mann"),
+        ("Mënner", "Mond"),
+        ("kritt", "kréien")
+    ],
+)
+def test_lb_lemmatizer_lookup_assigns(lb_tokenizer, string, lemma):
+    tokens = lb_tokenizer(string)
+    assert tokens[0].lemma_ == lemma
```

Comment:
See my comment on the lemma lookups – these tests should move to the [`spacy-lookups-data` repo](https://github.com/explosion/spacy-lookups-data) as well.

Reply:
I made the changes in `spacy-lookups-data`.